notebooks/isc/ISC.ipynb

```diff
@@ -92,8 +92,7 @@
    "source": [
     "# Download and extract example data from Zenodo\n",
     "!wget https://zenodo.org/record/4300904/files/brainiak-aperture-isc-data.tgz\n",
```

Comment:
Should we have `--no-clobber` so the file is not downloaded again? Same goes for any other download code in notebooks.